docs/index-management/index-reindexing.md

```diff
@@ -0,0 +1,215 @@
+# Index Reindexing
```

Comment:
_Index Reindexing_ sounds strange. Why not just _Reindexing_?

Reply:
If you only write reindexing, what do we want to reindex, e.g. a file system.